src/components/_classes/input/Input.js

```diff
@@ -222,7 +222,7 @@ export default class Input extends Multivalue {
   }
 
   getValueAsString(value, options) {
-    return super.getValueAsString(this.getWidgetValueAsString(value, options), options);
+    return super.getValueAsString(value, options);
```

Comment:
Isn't this the same thing as just removing the method all together?

Reply:
Yes) You are right. Just deleted it.